apps/design/backend/package.json

```diff
@@ -17,7 +17,7 @@
     "lint": "pnpm type-check && eslint .",
     "lint:fix": "pnpm type-check && eslint . --fix",
     "pre-commit": "lint-staged",
-    "start": "node ./build/index.js",
+    "start": "node ./build/index.js & node ./build/worker/index.js",
```

Comment:
Did some testing to verify that ctrl-c closes both processes. Presumably handled by pnpm